Sources/NostrSDK/Events/ReactionEvent.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  File.swift
```

Comment:
It's annoying that this is here by default, but let's update it I guess.

Reply:
Good catch. I'll update it before merging.